src/sql/base/browser/ui/table/table.ts

```diff
@@ -49,6 +49,10 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 	private _onClick = new Emitter<ITableMouseEvent>();
 	public readonly onClick: Event<ITableMouseEvent> = this._onClick.event;
 
+	//my addition
```

Comment:
Are these comments for you? I'd just remove them - they don't seem to provide anything useful at the moment